docs/tutorials/blog.md

```diff
@@ -203,12 +203,8 @@ type Post = {
   title: string;
 };
 
-type LoaderData = {
-  posts: Array<Post>;
-};
-
 export const loader = async () => {
```

Comment:
I would recommend turning this from a `export const` to an `export async function`, also you need to use `LoaderArgs`, but instead of `{}: LoaderArgs` you can use `args: LoaderArgs` or something like that, that way ESLint won't complain.

```ts
export async function loader(args: LoaderArgs) {
  // ...
}
```

Read release notes for [v1.6.5](https://github.com/remix-run/remix/releases/tag/remix%401.6.5) for more info.